detox/detox.d.ts

```diff
@@ -1112,6 +1112,31 @@ declare global {
              * web.element(by.web.tag('mark'))
              */
             tag(tagName: string): WebMatcher;
+
+            /**
+             * (iOS Only) Find an element on the DOM tree by its value
+             * @param value
+             * @example
+             * web.element(by.web.value('hello'))
+             */
+            value(value: string): WebMatcher;
+
+            /**
+             * (iOS Only) Find an element or secured element on the web-view by its accessibility label.
+             * @param text
+             * @example
+             * web.element(by.web.label('Submit')).asSecured()
+             * web.element(by.web.label('Submit'))
+             */
+            label(text: string): MaybeSecuredWebMatcher;
+
+            /**
+             * (iOS Only) Find a secured element on the web-view by its accessibility type.
+             * @note Secured-Web APIs are still in experimental phase and are subject to changes in the near future.
+             * @example
+             * web(by.web.type('textField')).asSecured()
+             */
+            type(type: string): SecuredWebMatcher;
```

Comment:
How do you plan to solve issue that iOS and Android types are likely to be very different?

`device.getPlatform() === 'ios' ? ... ` or ... how?

Reply:
This uses the accessibility types from the iOS system level (see XCUITest, Accessibility Inspector). I believe the best approach will be to understand what heuristics we can use for Android accessibility types to classify different elements and come up with new shared types. At the moment, I prefer to simply expose the iOS types to avoid premature optimization. This is an experimental API, so once we have a good candidate to replace it, we'll do it.